index.html

```diff
@@ -10,7 +10,7 @@
   </head>
   
   <body>
-  
+  <h1>Welcome to Potionfy! Enjoy your stay.</h1>
```

Comment:
Could you remove the 'Enjoy your stay.' string and make the 'Welcome to Potionfy!' string red?